client/store/bikes.js

```diff
@@ -56,8 +62,15 @@ const getOneBike = bike => ({type: GET_SINGLE_BIKE, bike})
      }
  }
 
+ export const filter = categoryVal => {
+   return async dispatch => {
```

Comment:
not sure we need async in this -- is any of this ultimately calling axios/waiting for backend info?